<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>958</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/cryptonote</normaloff>:/images/cryptonote</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="m_start">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_openWalletButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>275</width>
            <height>75</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Open Wallet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="m_createNewWalletButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>75</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create New Wallet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="m_restoreWalletFromKeysButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>425</width>
            <height>75</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Restore Wallet from Keys</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="WalletGui::TransactionsFrame" name="m_transactionsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::SendFrame" name="m_sendFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::OverviewFrame" name="m_overviewFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1550</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_wallets">
     <property name="title">
      <string>Recent wallets</string>
     </property>
    </widget>
    <addaction name="menuRecent_wallets"/>
    <addaction name="m_openWalletAction"/>
    <addaction name="m_createNewWalletAction"/>
    <addaction name="m_restoreWalletFromKeysAction"/>
    <addaction name="m_closeWalletAction"/>
    <addaction name="separator"/>
    <addaction name="m_exitAction"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="m_changePasswordAction"/>
    <addaction name="m_showPrivateKeys"/>
    <addaction name="m_connectionSettingsAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m_aboutCryptonoteAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_overviewAction"/>
   <addaction name="m_sendAction"/>
   <addaction name="m_transactionsAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="m_overviewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <underline>false</underline>
    </font>
   </property>
  </action>
  <action name="m_sendAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="m_transactionsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </action>
  <action name="m_exitAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="m_createNewWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create new wallet</string>
   </property>
  </action>
  <action name="m_openWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open wallet</string>
   </property>
  </action>
  <action name="m_changePasswordAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
   <property name="toolTip">
    <string>Change password</string>
   </property>
  </action>
  <action name="m_aboutCryptonoteAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="m_restoreWalletFromKeysAction">
   <property name="text">
    <string>Restore wallet from keys</string>
   </property>
  </action>
  <action name="m_showPrivateKeys">
   <property name="text">
    <string>Show private keys</string>
   </property>
  </action>
  <action name="m_connectionSettingsAction">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="m_closeWalletAction">
   <property name="text">
    <string>Close wallet</string>
   </property>
  </action>
  <action name="m_recentWalletsAction">
   <property name="text">
    <string>Recent wallets</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::OverviewFrame</class>
   <extends>QFrame</extends>
   <header>gui/OverviewFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::SendFrame</class>
   <extends>QFrame</extends>
   <header>gui/SendFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::TransactionsFrame</class>
   <extends>QFrame</extends>
   <header>gui/TransactionsFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_transactionsAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_transactionsFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createNewWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNewWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_restoreWalletFromKeysAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>restoreWalletFromKeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_changePasswordAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryptWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutCryptonoteAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_showPrivateKeys</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPrivateKeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_connectionSettingsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openConnectionSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_sendFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_overviewAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_overviewFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openWalletButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createNewWalletButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNewWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_restoreWalletFromKeysButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>restoreWalletFromKeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createNewWallet()</slot>
  <slot>openWallet()</slot>
  <slot>encryptWallet()</slot>
  <slot>about()</slot>
  <slot>showPrivateKeys()</slot>
 </slots>
</ui>
